src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -116,8 +116,8 @@ CHIP_ERROR writeFabricsIntoFabricsListAttribute()
     uint8_t fabricIndex = 0;
     for (auto & pairing : GetGlobalFabricTable())
     {
-        NodeId nodeId               = pairing.GetNodeId();
-        uint64_t fabricId           = pairing.GetFabricId();
+        NodeId nodeId               = pairing.GetPeerId().GetNodeId();
+        uint64_t fabricId           = pairing.GetUncompressedFabricId();
```

Comment:
Suggest using `GetRawFabricId()` as a better name.

Also, `writeFabric` is missing the root public key. It still seems like the code expects a single fabric space, and fabric ID collisions are still possible.

Reply:
`GetRawFabricId()` sounds good.

`writeFabric` needs to be updated. The `FabricDescriptor` structure is out of date compared to the latest spec.
It's also missing `FabricIndex`, and ordering of the fields is not correct. I am planning a separate PR for this part of the code.

Reply:
https://github.com/project-chip/connectedhomeip/issues/9194